manifests/custom_profile_content.pp

```diff
@@ -14,7 +14,7 @@
 define authselect::custom_profile_content (
   String               $content,
   Pattern[
-    /^\/etc\/authselect\/custom\/[^\/]+\/[^\/]+$/
+    /(^\/etc\/authselect\/custom\/[^\/]+\/[^\/]+$)|(^\/usr\/share\/authselect\/vendor\/[^\/]+\/[^\/]+$)/
```

Comment:
Wouldn't that just be
```
/^\/etc\/authselect\/(custom|vendor)\/[^\/]+\/[^\/]+$/
```

Reply:
Its /usr/share/authselect/vendor vs /etc/authselect/custom, I actually mistakenly tried that first and was scratching my head when stuff didn't work on my machine.

Reply:
Ah, sorry, I totally missed that.  In that case, to make it a bit more obvious, I would do this:
```puppet
  Pattern[
    /^\/etc\/authselect\/custom\/[^\/]+\/[^\/]+$/,
    /^\/usr\/share\/authselect\/vendor\/[^\/]+\/[^\/]+$/
  ]                    $path   = "/etc/authselect/custom/${name}",

Reply:
Done!